Plugins/BackgroundFetch/BackgroundFetchPlugin.cs

```diff
@@ -126,7 +126,7 @@ private void RecreateObservable()
                                       string msg;
                                       try
                                       {
-                                          msg = _currentGitUiCommands.GitModule.GitExecutable.GetOutput(args);
+                                          msg = _currentGitUiCommands.GitModule.GitExecutable.Execute(args).AllOutput;
```

Comment:
```suggestion
                                          msg = _currentGitUiCommands.GitModule.GitExecutable.Execute(args).StandardError;
```

Reply:
Agreed. But one step more. We need to run through all use of all output and do the same. Does it need to be all output or std based on what we look for.  Look at the git source code to track what output we look for is written where. It would prevent confusion. Down the road to put good comments in the use of error to mention git writes blaa to error so we read error instead of std and such.

Reply:
@gerhardol I just pushed and tested your suggestion, and I also added some comments.
To me this fix is enough to solve this specific issue. I'm not sure if you want to include the full review of all output use in the scope of this PR or not, because then I'm afraid I don't have enough expertise to do that.

Reply:
Good work, hanging on.
Now this is handled not just as a fix but with a known reason and explanation.
This could be included in a 4.0.3 (if there is one), a separate PR there later.

@vbjay Also thanks for hanging on. There were so many opinions that it was hard to find the core facts. (If I tried to get all data right before posting anything, a lot less would be done why asked in the issue as a question.)
There were some investigations before #9488, but a full check would take too much time.
In a community project like GE, the members need to be involved.